int32 drone_id

int8[] grid_ids
float64 recent_attempt_time
float64 stamp

# only used for simulation
float32[] pos
float32[] vel
float32 yaw

# FAME
geometry_msgs/Point goal_posit
int8 role
